lambda/entry_generic.go

```diff
@@ -0,0 +1,52 @@
+//go:build go1.18
+// +build go1.18
```

Comment:
+build is the pre-go1.17 syntax. `go fmt` automatically adds the go:build comment so I've included both